src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1118,33 +767,33 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
         {
             //  Detect CitizenData whose EntityCitizen no longer exist in world, and clear the mapping
             //  Consider handing this in an ChunkUnload Event instead?
-            citizens.values()
+            citizenManager.getCitizens()
```

Comment:
This whole section should probably be moved into the citizen manager.

Reply:
Yeah probably like in the buildingManager

Reply:
Correct